src/BarChart/BarChart.tsx

```diff
@@ -377,7 +377,7 @@ export const BarChart: FC<Partial<BarChartProps>> = ({
               element={brush}
               height={chartHeight}
               width={chartWidth}
-              scale={xScale}
+              scale={yScale}
```

Comment:
so, this is what it boiled down to afaict .. but @amcdnl lmk if I'm missing something major here

Reply:
The brush should be on the xscale - so I don't think this is the right fix.